Add Healthchecks

waywardone opened the pull request:
I use healthchecks successfully without traefik so this exact PR hasn't been tested, so please update as needed if I got any of those parts wrong.

robflate commented:
What is the Environment Variable `BASE_URL`? I can't see it in the official docs https://docs.linuxserver.io/images/docker-healthchecks

waywardone commented:
Healthchecks uses a `local_settings.py` with a longer list of environment variables mentioned in the [Configuration section of the project README](https://github.com/healthchecks/healthchecks). That being said, it seems like `BASE_URL` is generated in my running instance and doesn't need to be specified. Removed it.

I switched back to environment for the `SUPERUSER_PASSWORD`. I tried secrets with the required declarations and `FILE__` prefix and couldn't get it to work.
